src/unix/linux_like/android/mod.rs

```diff
@@ -1277,6 +1277,8 @@ pub const PTRACE_SETOPTIONS: ::c_int = 0x4200;
 pub const PTRACE_GETEVENTMSG: ::c_int = 0x4201;
 pub const PTRACE_GETSIGINFO: ::c_int = 0x4202;
 pub const PTRACE_SETSIGINFO: ::c_int = 0x4203;
+pub const PTRACE_GETREGSET: ::c_uint = 0x4204;
+pub const PTRACE_SETREGSET: ::c_uint = 0x4205;
```

Comment:
We've declared other `PTRACE_*`s as `c_int`, they may cause a confusion.

Reply:
Woops. Copy paste error. Fixed.